scripts/release/publish-docs-artifacts.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Publish material2 docs assets to the material2-docs-content repo
+# material.angular.io will pull from this assets repo to get the latest docs
+
+cd "$(dirname $0)/../../"
+root="$(pwd)"
+
+docsPath="./dist/docs"
+tempRepoPath="./dist/tempRepo"
+repoUrl="https://github.com/angular/material2-docs-content/"
+
+# If the docs directory is not present, abort
+if [ ! -d $docsPath ]; then
+  echo "Material docs not generated. Aborting..."
+  exit 0
```

Comment:
Why not returning an exit code `1` here? If Travis turns red we can see that there are issues with the docs.